docs/config.toml

```diff
@@ -87,7 +87,7 @@ version_menu = "Releases"
   url = "https://edge.docs.radapp.io"
 
 [[params.versions]]
-  version = "latest"
+  version = "v0.31"
```

Comment:
Before we merge this we need to update the release script, as that has logic to automatically upgrade this version number on every release.

Reply:
Do you mean this? https://github.com/radius-project/docs/blob/v0.31/.github/workflows/release.yaml

Reply:
This script specifically: https://github.com/radius-project/docs/blob/v0.31/.github/scripts/release-docs.sh

Reply:
Modified the script and tested it out locally, please see above for the test script and the result.
For a more specific call out the line:

```
awk -v REPLACEMENT="${VERSION_STRING_REPLACEMENT}" '{gsub(/^[ \t]*version = "latest"/, REPLACEMENT); print}' docs/config.toml > docs/config.toml.tmp
```
Looks for a line that contains `version = "latest"` while accepting white spaces or tabs but not params such as `tag_version` which would be caught without the strict expression